test/e2e/alertmanager_test.go

```diff
@@ -509,3 +510,48 @@ func TestAlertmanagerDisabling(t *testing.T) {
 		}
 	})
 }
+
+func TestAlertManagerHasAdditionalAlertRelabelConfigs(t *testing.T) {
+	const (
+		expectPlatformLabel      = "openshift_io_alert_source"
+		expectPlatformLabelValue = "platform"
+	)
+
+	type Alerts []struct {
+		Labels map[string]string `json:"labels"`
+	}
+
+	var alerts Alerts
+
+	err := framework.Poll(5*time.Second, time.Minute, func() error {
+		resp, err := f.AlertmanagerClient.Do("GET", "/api/v2/alerts", nil)
```

Comment:
we could also filter on alerts with `openshift_io_alert_source="platform"`. Similar to:

https://github.com/openshift/cluster-monitoring-operator/blob/1824f9c9a39f54734298dd10e5d20d42c8247995/test/e2e/alertmanager_test.go#L399-L402

Reply:
would we want to filter though? my understanding was the test should check that all alerts have the label applied?

Reply:
as long as no UWM alerts is firing, it should be fine indeed.

Reply:
well that is a good point actually, although we did put a fair amount of work in to ensure the tests are isolated and each run is cleaned up, so if it fails in its current format, then it probably highlights a different problem  :)